package-lock.json

```diff
@@ -6455,7 +6455,7 @@
     },
     "lru-cache": {
       "version": "4.0.0",
-      "resolved": "https://registry.npmjs.org/lru-cache/-/lru-cache-4.0.0.tgz",
+      "resolved": "http://registry.npmjs.org/lru-cache/-/lru-cache-4.0.0.tgz",
```

Comment:
`npm install` always wants to change this line. It seems _wrong_ to downgrade from `https` to `http`. However, since it looks to be a [registry issue](https://npm.community/t/npm-install-downgrading-resolved-packages-from-https-to-http-registry-in-package-lock-json/1818/16) with the package, I've included this change to avoid noise when running `npm install` locally.

Happy to split into a separate PR if you'd like.